src/Opdex.Platform.WebApi/Models/Requests/SnapshotFilterParameters.cs

```diff
@@ -0,0 +1,33 @@
+using Opdex.Platform.Common.Extensions;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Queries;
+using System;
+
+namespace Opdex.Platform.WebApi.Models.Requests
+{
+    public class SnapshotFilterParameters : FilterParameters<SnapshotCursor>
+    {
+        /// <summary>
+        /// The snapshot step interval.
+        /// </summary>
+        public Interval Interval { get; set; }
+
+        /// <summary>
+        /// Start time for which to retrieve snapshots.
+        /// </summary>
+        public DateTime StartTime { get; set; }
+
+        /// <summary>
+        /// End time for which to retrieve snapshots.
+        /// </summary>
+        public DateTime EndTime { get; set; }
```

Comment:
Probably nothing, curious on if you think `StartDateTime` and `EndDateTime` might be clearer for clients, probably doesn't matter at all, just a thought.

Reply:
I agree, makes it clearer. Updated them